patch_geopandas.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+sed -i "s/from pandas.core.internals import Block, NonConsolidatableMixIn, \
+BlockManager/from pandas.core.internals.blocks import Block, \
+NonConsolidatableMixIn\nfrom pandas.core.internals.managers import \
+BlockManager/g" /opt/conda/lib/python3.6/site-packages/geopandas/_block.py
```

Comment:
??!?!

Can you add a comment pointing to the issue/comment where you found this and under what circumstances this should be removed?

Reply:
Haha yeah that's the appropriate reaction to this.

See [here](https://github.com/geopandas/geopandas/issues/473#issuecomment-465060849)

It's an issue w/ the fact that dev branch of geopandas hasn't been updated in a year. I think it can probably removed whenever we get a version of geopandas that can do vectorized operations (which looks like it will no longer be the merging of their cython branch but rather a new branch that leverages a package called `pyGEOS`)